### DISCUSSION TITLE
Pytorch to onnx and use with opencv-dnn?

### DISCUSSION THREAD

**Discussion by RizwanMunawar** [ultralytics/yolov5#4404] [Q&A]
I want to run yolov5 with opencv dnn in C++, for this, I have converted PyTorch model to onnx, from this link`https://github.com/ultralytics/yolov5/blob/master/export.py`,But that onnx is not working with opencv dnn module.
Anyhelp would be appreciated.

**Comment by glenn-jocher** [comment]
@RizwanMunawar if you are having issues with opencv you should raise them directly with the package authors.
